tests/Domain/PersonTest.php

```diff
@@ -11,25 +11,7 @@
 
 class PersonTest extends TestCase
 {
-    public function personProvider(): array
-    {
-        return [
-            [1, 'bill.gates', 'Bill', 'Gates'],
-            [2, 'steve.jobs', 'Steve', 'Jobs'],
-            [3, 'mark.zuckerberg', 'Mark', 'Zuckerberg'],
-            [4, 'evan.spiegel', 'Evan', 'Spiegel'],
-            [5, 'jack.dorsey', 'Jack', 'Dorsey'],
-        ];
-    }
```

Comment:
deleted dataprovider as none of the tests actually used any of the data it provides via their arguments.